app/src/main/res/layout/application_list_item.xml

```diff
@@ -8,39 +8,17 @@
     <android.support.v7.widget.CardView
         android:id="@+id/card_view"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_height="@dimen/image_height"
         android:layout_gravity="center"
         android:layout_margin="@dimen/card_margin"
         card_view:cardCornerRadius="@dimen/card_radius">
 
-        <LinearLayout
+        <ImageView
```

Comment:
I suggest don't copy the complete image in the card view else separate the text from the background and write the title and description in two textview with an image in the background. That way we will have similar font in all the card views.